functions/Remove-DbaAgentAlertCategory.ps1

```diff
@@ -0,0 +1,117 @@
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags: Agent, Alert, AlertCategory
+        Author: Patrick Flynn (@sqllensman)
+
+        Website: https://dbatools.io
+        Copyright: (c) 2018 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaAgentAlertCategory
+
+    .EXAMPLE
+        PS C:\> Remove-DbaAgentAlertCategory -SqlInstance sql1 -Category 'Category 1'
+
+        Remove the alert category Category 1 from the instance.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaAgentAlertCategory -SqlInstance sql1 -Category Category1, Category2, Category3
+
+        Remove multiple alert categories from the instance.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaAgentAlertCategory -SqlInstance sql1, sql2, sql3 -Category Category1, Category2, Category3
+
+        Remove multiple alert categories from the multiple instances.
+
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = "Low")]
+    param (
+        [parameter(Mandatory, ValueFromPipeline)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [ValidateNotNullOrEmpty()]
+        [string[]]$Category,
+        [switch]$Force,
+        [switch]$EnableException
+    )
+    begin {
+        if ($Force) {$ConfirmPreference = 'none'}
+    }
+    process {
+
+        foreach ($instance in $SqlInstance) {
+            # Try connecting to the instance
+            try {
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+            } catch {
+                Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+            }
+
+            # Loop through each of the categories
+            foreach ($cat in $Category) {
+
+                # Check if the alert category exists
+                if ($cat -notin $server.JobServer.AlertCategories.Name) {
+                    Stop-Function -Message "Alert category $cat doesn't exist on $instance" -Target $instance -Continue
+                }
+
+                # Remove the category
+                if ($PSCmdlet.ShouldProcess($instance, "Removing the alert category $Category")) {
+                    try {
+                        # Get the category
+                        $currentCategory = $server.JobServer.AlertCategories[$cat]
+
+                        Write-Message -Message "Removing alert category $cat" -Level Verbose
+
+                        $currentCategory.Drop()
+                    } catch {
+                        Stop-Function -Message "Something went wrong removing the alert category $cat on $instance" -Target $cat -Continue -ErrorRecord $_
+                    }
+
+                } #if should process
```

Comment:
Thanks so much for the new commands! 

Some dbatools commands (and the ones that you based this off) still comment brackets in this manner, but it's only because I haven't cleaned them up yet ;) With common editors like VS Code, it's easy to collapse so comments like these are unnecessary.

In general, we want to reserve comments to explain why something unusual or complex is being done, instead of what is being done.